CountryPicker/CountryPicker/CountryManager.swift

```diff
@@ -71,10 +71,21 @@ open class CountryManager {
     internal var filters: Set<CountryFilterOption> = [.countryName]
     
     
-    private init() {}
+    public init() {}
     
-    func loadCountries() throws {
+    open func loadCountries() throws {
+        let countryCodes = try! getCountryCodes()
         
```

Comment:
Shouldn't we throws backs instead of generating crash here.
`let countryCodes = try! getCountryCodes()`
uses 
`let countryCodes = try getCountryCodes()` because `loadCountries` can throw itself